src/content/blog/filecoin-nv19-lightning-upgrade.md

```diff
@@ -34,6 +34,6 @@ The successful launch of the Lightning Upgrade represents the collective efforts
 
 The nv19 Lightning Upgrade represents a remarkable achievement, made possible by the dedication and innovative spirit of the entire Filecoin community and implementation teams. This milestone sets the tone for the future of the network, pushing the boundaries of decentralized storage and reaffirming Filecoin's position at the forefront of the industry.
 
-Looking ahead, governance and community engagement remain critical components of designing, driving and instantiating robust governing systems, tools, and methodologies. Filecoin Foundation through the governance team has put together monthly Community Governance Calls. You can register and be part of this process [here](https://fil-org.zoom.us/meeting/register/tZMkf-2qpjIqEt3nkeKh_7f7_F6wPm76zbTw).
+Looking ahead, governance and community engagement remain critical components of designing, driving and instantiating robust governing systems, tools, and methodologies. Filecoin Foundation through the governance team has put together monthly Community Governance Calls.
```

Comment:
Ideally we would like to the Upcoming Calls sections - but we don't have anchor links in place yet.

```suggestion
Looking ahead, governance and community engagement remain critical components of designing, driving and instantiating robust governing systems, tools, and methodologies. Filecoin Foundation through the governance team has put together monthly Community Governance Calls. You can register and be part of this process [here](https://fil.org/governance).
```